src/platforms/dialogflow/google/GoogleAssistantSession.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Context,
+  Contexts,
+  DialogflowConversation,
+  Parameters,
+} from "actions-on-google";
+import * as _ from "lodash";
+import { IBag, IVoxaSession } from "../../../VoxaEvent";
+
+export class DialogflowSession implements IVoxaSession {
```

Comment:
any reason to not just use that? why do we need specific session classes for facebook and assistant?

Reply:
The `GoogleAssistantSession` class uses objects from the `actions-on-google` package. While `FacebookSession` simply controls sessions in one of the contexts of Dialogflow request, which is what the action-on-google package does as well, but I just didn't want to mix things up between them. I think that maybe I can leave the FacebookSession code in DialogflowSession to be used not only by Facebook but also for any other generic platform like Telegram. And since GoogleAssistant has its own manager, I can make it implement the DialogflowSession class and overwrite that method.

Thoughts on that ?